requirements.txt

```diff
@@ -3,7 +3,7 @@ cycler==0.10.0
 kiwisolver==1.1.0
 matplotlib==3.1.3
 numpy==1.18.1
-opencv-python==3.3.1
+opencv-python==3.4.2.17
 pandas==1.0.2
 pyparsing==2.4.6
 python-dateutil==2.8.1
```

Comment:
scikit-image is one of the requirements. I've documented tested list of requirements here (https://github.com/czbiohub/pysero/blob/master/docs/installation.md)

Reply:
Scikit-image was already in the requirements file, but scipy wasn't so I added that.
I don't see any uses of cython or xlrd in the entire repo, should they still be in requirements?